NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -207,22 +231,25 @@ def begin(self,
          self._time_low = None
          self._time_high = None         
       
+      self.__run_table = None
       if select_runs:
          if run_table_path is None:
-            global imported_runtable
-            if imported_runtable:
+            try:
+               from rnog_data.runtable import RunTable
                self.logger.debug("Access RunTable database ...")
                try:
                   self.__run_table = RunTable().get_table()
                except:
                   self.logger.error("No connect to RunTable database could be established. "
                                     "Runs will not be filtered.")
-                  imported_runtable = False
+            except ImportError:
+               self.logger.error("Import of run table failed. You will not be able to select runs! \n" 
+                     "You can get the interface from GitHub: git@github.com:RNO-G/rnog-data-analysis-and-issues.git")
          else:
+            import pandas
             self.__run_table = pandas.read_csv(run_table_path)
-            imported_runtable = True
```

Comment:
What do you mean? We typically call `begin(..)` only once right?

Reply:
I guess I'm still too used to the 'old' reader where I would pass it different input files directly as opposed to using the selection capabilities in the `begin` function. But say I was interested in analysing some specific runs or events for different stations (e.g. station 23 runs 1005-1034, station 11 runs 1911-1935, ...), wouldn't the most straightforward approach be to just call `begin` multiple times?

Anyway, I guess loading the run table multiple times will not be particularly performance critical even in those cases, especially if/when some kind of buffering gets implemented in `RunTable`. It just felt tidier to me to have this bit in an `__init__`, because it wouldn't change even if the run selection changes.

Reply:
You can just process data from several station in one go by passing a list with all those directories. Well I feel it is awkward to "split" the configuration of the module into two functions but that might be due to the fact that I am not used to the use case you described or did not quite understand the purpose of `begin` and `init` in the first place.

Thinking about it I might be onboard with the idea because it could be more readable splitting the runs selection and relevant parameters from the other once... what do you think? However, I would rather use the new PR for this and merge this one here asap.